packages/components/popper/src/content.vue

```diff
@@ -112,7 +111,15 @@ const ariaModal = computed<string | undefined>(() => {
   return role && role.value === 'dialog' ? 'false' : undefined
 })
 
-const createPopperInstance = ({ referenceEl, popperContentEl, arrowEl }) => {
+const createPopperInstance = ({
+  referenceEl,
+  popperContentEl,
+  arrowEl,
+}: {
+  referenceEl: Measurable
+  popperContentEl: HTMLElement
+  arrowEl: HTMLElement | undefined
+}) => {
```

Comment:
**Suggestion:** can we move these types into a local type such as `CreatePopperInstanceParams`?

Reply:
Updated.